### DISCUSSION TITLE
【广告】凯木都市轻型车

### DISCUSSION THREAD

**Discussion by xiri61** [share121/inter-knot#549] [其他]
![微信图片_20240810111542_upscayl_4x_RealESRGAN_General_x4_v3](https://github.com/user-attachments/assets/430fc826-e8f3-49c8-a7c3-ba24abdde9b5)
【广告】凯木都市轻型车，平静不平凡！你是否为车辆价格过高而苦恼？你是否为出行停车困难而心烦？ CALM—凯木都市轻型车，代步新传说，平静不平凡！ 现在购买， 尊享24期免息分期付款！（详情请咨询奥佛汽车线下门店）

**Comment by 1122246** [comment]
避雷！！！！！车轮噪音极大，售后服务极差！！

**Comment by Codeflame1** [comment]
楼上的黑子差不多得了

**Comment by kaijinti** [comment]
二十四期免息？一个三蹦子？

**Comment by KEARUM** [comment]
每次在街上看到凯木转弯，我都担心它会侧翻....

**Comment by caizibook** [comment]
24期免息？代价呢，你这车不会是溢价严重的吧？

**Comment by Liyang0204** [comment]
未付款，期待发货

**Comment by Hongxuli2006** [comment]
笑死，吸一吸赛博龙脉
